### PR TITLE
Allow rift explorer to work even if the new Riot Client exists

### PR DESCRIPTION
**Pull request by Morilli**
By just bypassing it 🤷‍♀

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "yaml": "^1.6.0"
   },
   "devDependencies": {
-    "electron": "^6.0.5",
+    "electron": "^5.0.10",
```
Comment (RayZz-) [inline package.json]: I don't understand why this is even adjusted

Comment (Pupix) [inline package.json]: why go back to electron 5?

File: util.js [hunk 2]
```diff
@@ -39,9 +39,10 @@ async function duplicateSystemYaml() {
     }
 
     const file = await fs.readFile(originalSystemFile, 'utf8');
-    const fileParsed = yaml.parse(file);
-    
-    fileParsed.enable_swagger = true;
+    const fileParsed = yaml.parseDocument(file);
+
+    fileParsed.delete("riotclient");
```
[the remark this answers is not in the document]
Comment (Morilli) [reply util.js]: From what I've seen, installations that have the riotclient enabled have a part called ``riotclient`` at the end of the system.yaml file: 
image
Disabling (or just removing) that makes it possible to launch the ``LeagueClient.exe`` without automatically starting the new Riot Client instead.

File: package.json (hunk 1 above)
Comment (Morilli) [reply package.json]: See the specific commit I made for this change. The ``dialog.showMessageBox`` in ``app.js`` in line 79 does not block the Rift Explorer process in electron 6, causing the client to be restarted immediately the moment the message box pops up without waiting for the user to click "OK". This is effectively a workaround for an error either on electron or Rift Explorer's end, but as going back one electron version fixes it I believe that to be a bug in electron.

If you find there to be a cleaner fix for this issue, I'd be happy to apply that, but for the time being I think this is the best.

Comment (RayZz-) [reply package.json]: This isn't a bug. however this issue has been resolved in another PR. so downgrading isn't really necessary.

Comment (Morilli) [reply package.json]: @RayZz- Yeah, I saw that now. Apparently the ``showMessageBox`` function was renamed, and in its place a different function was created that runs asynchronously. The documentation still states that this function blocks the process though, so these two occurrences are probably the root cause of the issue here.

@Pupix Is it okay if I force-push to have just one commit remaining?

Comment (RayZz-) [reply package.json]: it was renamed to `showMessageBoxSync`

File: util.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (RayZz-) [reply util.js]: My only one concern being when the riot client becomes the main client will this still work?

[the remark this answers is not in the document]
Comment (Morilli) [reply util.js]: Main client? From what I understand the new client is supposed to be nothing more than a launcher for the "real" client, enabling some "stay logged in" support and maybe other stuff; haven't really looked into it much. So as it stands, I don't see any issues here.

Of course, if matters change in the future (in the end, the riot client isn't even available freely yet), things might be changed again.

File: package.json (hunk 1 above)
Comment (Morilli) [reply package.json]: Have reverted this commit. Will leave it to your PR.

**Comment by Pupix** [comment]
I just went ahead and made these changes while I was editing other stuff.

**Comment by Kixej** [comment]
Bro can u help me i have waiting for league please help me

**Pull request closed without merging**